Code/LauncherUnified/launcher_generator.cmake

```diff
@@ -22,7 +22,7 @@ foreach(project_name project_path IN ZIP_LISTS LY_PROJECTS_TARGET_NAME LY_PROJEC
         o3de_read_json_key(project_name ${project_real_path}/project.json "project_name")
         message(WARNING "The project located at path ${project_real_path} has a valid \"project name\" of '${project_name}' read from it's project.json file."
             " This indicates that the ${project_real_path}/CMakeLists.txt is not properly appending the \"project name\" "
-            "to the LY_PROJECTS_TARGET_NAME global property. Other configuration errors might occur")
+            "to the O3DE_PROJECTS_NAME global property. Other configuration errors might occur")
```

Comment:
LIke I understand every individual word in this error message but I don't know what it wants me to do or what it means in general.  Its telling me something is valid, but something is wrong in CMakeLists.txt and there will be problems..

Reply:
I'll clean up the error message